webserver/external/messybrainz.py

```diff
@@ -0,0 +1,36 @@
+import requests
+import ujson
+
+def get_messybrainz_id(querydata):
+    """
+    This function queries messybrainz to get a msid according to the 'artist/title' present in the item metadata
+    Args:
+        querydata: A dictionary containing at least artis and song title
+
+    Returns:
+        msid: string messybrainz id -> if it is found in the messybrainz database
+        None: if no reference is fount in messybrainz database
+
+    """
+    msurl = 'https://messybrainz.org/submit'
```

Comment:
An additional feature that we need to add here is user authentication for messybrainz. Our preference is that only specific services can access it, so we should have some sort of authentication system. 
If you want, you could do this yourself. It's a related feature, and would be a good introduction into setting up and working on another codebase. I can write a ticket describing what needs to be done.
If you would prefer to continue working on this part of acousticbrainz, let me know and I'll make the necessary changes in messybrainz for you.